### DISCUSSION TITLE
failed to update CodeWhisperer

### DISCUSSION THREAD

**Discussion by sahilparikh98** [aws/codewhisperer-command-line-discussions#205] [Support ticket] [answered]
### Checks

- [X] I ran `q doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `q restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/q-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

CodeWhisperer should update and restart

### Actual behaviour

CodeWhisperer fails to update and start

### Steps to reproduce

<img width="347" alt="Screenshot 2024-06-10 at 1 05 29 PM" src="https://github.com/aws/q-command-line-discussions/assets/12001573/14b700c7-a80c-4e53-92e4-c1200aad4797">
<img width="574" alt="Screenshot 2024-06-10 at 1 06 24 PM" src="https://github.com/aws/q-command-line-discussions/assets/12001573/f9d396e5-c485-4c66-a481-7551b3d68cdf">

### Environment

```toml
CODEWHISPERER-details:
  - 0.15.4
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 10
  - mem: 16.00 GB
os-info:
  - macOS 14.2.1 (23C71)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/sahilparikh
  - exe-path: /Users/sahilparikh/.local/bin/cw
  - install-method: brew
  - env-vars:
    - CWTERM_SESSION_ID: ac9db6a6b2e3491888c97ddb52d2d1e7
    - CW_SET_PARENT_CHECK: 1
    - CW_TERM: 0.15.4
    - PATH: /opt/homebrew/lib/ruby/gems/3.2.0/bin:/opt/homebrew/opt/ruby/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Library/Apple/usr/bin:/usr/local/munki:/Applications/iTerm.app/Contents/Resources/utilities:/Users/sahilparikh/.local/bin:/Users/sahilparikh/.fig/bin:/Users/sahilparikh/Library/Android/sdk/emulator:/Users/sahilparikh/Library/Android/sdk/tools:/Users/sahilparikh/Library/Android/sdk/tools/bin:/Users/sahilparikh/Library/Android/sdk/platform-tools
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.googlecode.iterm2
```

**Comment by grant0417** [comment, marked as answer]
Hey @sahilparikh98 run the following steps:
1. Delete the "CodeWhisperer.app": `rm -rf /Applications/CodeWhisperer.app`
2. Install "Amazon Q.app" from this link: https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-getting-started-installing.html

**Reply by venkateshsys** [reply]
It worked, thanks @grant0417
